conan/tools/cmake/cmakedeps/cmakedeps.py

```diff
@@ -112,3 +113,35 @@ def _generate_files(self, require, dep, ret, find_module_mode):
         # file is common for the different configurations.
         if not os.path.exists(config.filename):
             ret[config.filename] = config.render()
+
+    def set_property(self, dep, prop, value):
+        self._properties.setdefault(dep, {}).update({prop: value})
+
+    def get_property(self, prop, dep):
+        try:
+            return self._properties[dep].get(prop) or dep.get_property(prop)
```

Comment:
``cmakedeps.set_property(..., property_name, None)`` or ``value=""`` too) will still fallback to the dependency value? I think there should be a way to invalidate/clear the upstream value too. So as long as it doesn't ``KeyError``, I'd respect the local value?